### DISCUSSION TITLE
How to save REST response data?

### DISCUSSION THREAD

**Discussion by averrows** [Budibase/budibase#3688] [Help] [answered]
I want to create login flow in budibase with my own database with following requirements:
- I'm using REST to connect to my backend 
- I will post email and password to the server
- Server will authenticate me and sending back JWT

My question is:
- Can I save the JWT to some kind of variable? If yes, how to do it?

**Comment by mjashanks** [comment, marked as answer]
Hi @averrous-s 

You cannot currently do this with Budibase. However, at the end of December you will be able to do this. We are currently working on a big upgrade to our REST connector: 

https://github.com/Budibase/budibase/issues?q=is%3Aissue+is%3Aopen+%22REST+Connector+V2%22

**Reply by averrows** [reply]
Thanks for the info

**Reply by vaibhav93** [reply]
@mjashanks Is it possible now ? If yes, how. Thanks!

**Reply by acseven** [reply]
Check here if it helps:
https://github.com/Budibase/budibase/discussions/7159#discussioncomment-3356509
